spec/form_spec.cr

```diff
@@ -203,6 +203,14 @@ describe "LuckyRecord::Form" do
       form.name.value.should eq "Joe"
       form.name.errors.should eq ["wrong"]
     end
+
+    it "Field#value returns nil on empty strings" do
```

Comment:
Maybe extract a field_spec.cr spec so any future field specific tests can go there. What do you think?

Reply:
Sure, I was thinking the same but I didn't since there were only two specs.